src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborWriter.String.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+using System.Buffers.Binary;
+using System.Text;
+
+namespace System.Security.Cryptography.Encoding.Tests.Cbor
+{
+    internal partial class CborWriter
+    {
+        private static readonly System.Text.Encoding s_utf8Encoding = System.Text.Encoding.UTF8;
+
+        // Implements major type 2 encoding per https://tools.ietf.org/html/rfc7049#section-2.1
+        public void Write(ReadOnlySpan<byte> value)
```

Comment:
I feel like we started with everything just being overloads of `Write` in Utf8JsonWriter, then eventually we named all of the Write methods to each include their data type.

Maybe CBOR is simple enough that everything can be overloads; but I'm skeptical.

Reply:
What made you change approach in json?

Reply:
The thing that may have pushed it over the edge was the property writing overloads (`Write(string)` vs `Write(string, string)` / `Write(string, int)`).

Possibly that combined with the lack of symmetry to the reader; and lack of symmetry to existing stateful writers (like XmlWriter), which include the element type name in the method name.